### ISSUE TITLE
**SOLVED** Steam.exe [2023-11-15 16:31:50] Nothing to do

### ISSUE THREAD

**Issue by jorxster** [domschl/WinSteamOnMac#6]
Following the instructions, brew downloads and compiles game-porting-toolkit ok, but when under the "Continue" header, and it asks me to run this command:

`WINEPREFIX=~/Win10 `brew86 --prefix game-porting-toolkit`/bin/wine64 winecfg`

I get an error as I have no wine64 install inside game-porting-toolkit. How can I install wine64 here?

```
/usr/local/opt/game-porting-toolkit-compiler/lib:
total 101224
drwxr-xr-x  3 jordan.olson  admin    96B 13 Nov 19:23 clang/
drwxr-xr-x  5 jordan.olson  admin   160B 13 Nov 19:23 ./
-r--r--r--  1 jordan.olson  admin    21M 13 Nov 19:24 libLTO.dylib
-r--r--r--  1 jordan.olson  admin    29M 13 Nov 19:24 libclang.dylib
drwxr-xr-x  9 jordan.olson  admin   288B 13 Nov 19:24 ../
```

**Comment by domschl** [comment]
You probably skipped the 2nd command from:

```bash
brew86 tap apple/apple http://github.com/apple/homebrew-apple
brew86 -v install apple/apple/game-porting-toolkit
```

This would have installed the `wine` stuff.

**Comment by jorxster** [comment]
So, what I actually had to do was brew untap the other game-porting-toolkit (gcenx/wine/game-porting-toolkit)
and then was able to install the apple one. Previously I had let autocomplete to game-porting-toolkit-compiler, and that was my mistake.

NOW, I have Steam.exe installed and am trying to run it , but nothing shows up, no UI or GUI, it just seems to hang indefinitely?

...
[2023-11-15 16:31:49] Background update loop checking for update. . .
[2023-11-15 16:31:49] Checking for available updates...
[2023-11-15 16:31:49] Downloading manifest: https://cdn.steamstatic.com/client/steam_client_win32?t=1638232607
[2023-11-15 16:31:49] Manifest download: send request
[2023-11-15 16:31:49] Manifest download: waiting for download to finish
02a0: thread_get_state failed on Apple Silicon - faking zero debug registers
00fc:err:wininet:HTTP_ParseRfc1123Date unexpected time zone L"UTC"
00fc:err:wininet:HTTP_ParseRfc850Date unexpected month format L" Nov 2023 08:00:00 UTC"
00fc:err:wininet:HTTP_ParseDate unexpected date format L"Tue, 14 Nov 2023 08:00:00 UTC"
00fc:fixme:cryptnet:check_ocsp_response_info check responder id
[2023-11-15 16:31:50] Manifest download: finished
[2023-11-15 16:31:50] Download skipped by HTTP 304 Not Modified
[2023-11-15 16:31:50] Nothing to do

**Comment by jorxster** [comment]
SUCCESS !!!

UPDATE -- oh this is weird, after trying probably 4-6 times, I tried running it again and THIS time it opened up a UI!
I was able to log in and am attempting to download and install. And the first game I tried fired up flawlessly. A miracle.

Thank you so much **domschl** for your help and the guide.

Would be wise to put a note is there that if Steam ends with "Nothing to do" to just keep trying?
